components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinClientRequestAdapter.java

```diff
@@ -56,5 +56,15 @@ void onRequest(Exchange exchange, SpanCustomizer span) {
                 cache.reset();
             }
         }
+        
+        Object clientTagsObj = exchange.getProperty("camel.custom.tags");
+        if (clientTagsObj != null) {
+        	Map<String, String> clientTags = (Map)clientTagsObj;
+	        if (clientTags != null && !clientTags.isEmpty()) {
+	        	for (Map.Entry<String, String> tag: clientTags.entrySet()) {
+	        		span.tag("custom." + tag.getKey(), tag.getValue());
```

Comment:
Fix checkstyle, eg run `mvn compile -P sourcecheck` and fix the errors it report

Reply:
Fixed checkstyle issues and added unit test.